packages/graph/hash_graph/tests/integration/postgres/links.rs

```diff
@@ -234,44 +234,32 @@ async fn ordered_links() {
         .await
         .expect("could not create entity");
 
-    let person_c_metadata = api
-        .create_entity(person_c, person_type_id.clone(), None)
-        .await
-        .expect("could not create entity");
-
-    let _a_b_link = api
-        .create_ordered_link(
+    let link_entity_metadata = api
+        .create_link_entity(
+            Entity::empty(),
+            friend_link_type_id,
+            None,
             person_a_metadata.identifier().entity_id(),
             person_b_metadata.identifier().entity_id(),
-            friend_link_type_id.clone(),
-            2,
         )
         .await
         .expect("could not create link");
 
-    let _a_c_link = api
-        .create_ordered_link(
-            person_a_metadata.identifier().entity_id(),
-            person_c_metadata.identifier().entity_id(),
-            friend_link_type_id.clone(),
-            1,
-        )
-        .await
-        .expect("could not create link");
+    assert!(
+        !api.get_entity_links(person_a_metadata.identifier().entity_id())
+            .await
+            .expect("could not fetch link")
+            .is_empty()
+    );
 
-    let links_from_source = api
-        .get_entity_links(person_a_metadata.identifier().entity_id())
+    api.archive_entity(link_entity_metadata.identifier().entity_id())
         .await
-        .expect("could not fetch link");
+        .expect("could not remove link");
 
-    assert_eq!(
-        [
-            person_c_metadata.identifier().entity_id(),
-            person_b_metadata.identifier().entity_id(),
-        ][..],
-        links_from_source
-            .iter()
-            .map(|link| link.inner().target_entity())
-            .collect::<Vec<_>>()[..]
-    )
+    assert!(
+        api.get_entity_links(person_a_metadata.identifier().entity_id())
+            .await
+            .expect("could not fetch link")
```

Comment:
Like above, should this be the following as we're returning multiple?
```suggestion
            .expect("could not fetch links")
```